src/instructlab/lab.py

```diff
@@ -896,40 +895,23 @@ def convert(self, value, param, ctx) -> "torch.device":
 
 @cli.command()
 @click.option("--data-dir", help="Base directory where data is stored.", default=None)
-@click.option(
-    "--input-dir",
-    type=click.Path(),
-    show_default=True,  # TODO: set to None and change help message
-    help="Path to generated files to use as input.",
-)
 @click.option(
     "--skip-preprocessing",
     is_flag=True,
 )
 @click.option(
-    "--tokenizer-dir",
-    help="Base directory where tokenizer is stored.",
-    default=None,
-    show_default=True,
-)
-@click.option(
-    "--gguf-model-path",
-    help="Local directory where gguf model is stored.",
-    default=None,
-    show_default=True,
+    "--input-dir",
+    type=click.Path(exists=True, file_okay=False, dir_okay=True),
+    show_default=True,  # TODO: set to None and change help message
```

Comment:
Does this `TODO` need to be addressed?

Reply:
This isn't my TODO,it just looks like it is since I moved around the flags. I think this is out of scope for the PR, I'd hate to break input-dir.

Reply:
All good!